.travis.yml

```diff
@@ -0,0 +1,12 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
+- 2.2.3
+script:
+- bundle exec rubocop
+- bundle exec foodcritic -f any .
+- bundle exec rspec test/unit
```

Comment:
It'd be nice if we could do kitchen tests as well, we would need to figure out how to do kitchen tests on Travis (or another cloud based CI system) and ensure it's not brittle. Just not something we've done yet, but it'd be cool if we could.